src/go/rpk/pkg/api/admin/api_node_config.go

```diff
@@ -27,3 +31,9 @@ func (a *AdminAPI) GetNodeConfig(ctx context.Context) (NodeConfig, error) {
 
 	return nodeconfig, a.sendOne(ctx, http.MethodGet, "/v1/node_config", nil, &nodeconfig, false)
 }
+
+func (a *AdminAPI) RawNodeConfig(ctx context.Context) (RawNodeConfig, error) {
+	var nodeconfig RawNodeConfig
+
+	return nodeconfig, a.sendOne(ctx, http.MethodGet, "/v1/node_config", nil, &nodeconfig, false)
```

Comment:
Should this be GetRawNodeConfig? It's kinda looking like we don't have much consistency here actually 😅 

I also wonder if we should delete GetNodeConfig, or change it to NodeID()

Reply:
I've seen a couple of comments/discussions about not using Get<> for these handlers, but some handlers already have the Get prefix.

I personally prefer not to use the prefix, but open to discussion and will change it accordingly.

Reply:
Usually `Get` is not a Go thing, but in this package, it _somewhat_ corresponds to the HTTP method being issued.

That said, I'm fine to universally drop Get.